library_website/views.py

```diff
@@ -0,0 +1,62 @@
+from django.http import Http404
+from django.shortcuts import render
+from wagtail.images.models import Image
+from wagtail.models import Site
+
+
+def library_404_view(request, exception=Http404):
+
+    site_name = Site.find_for_request(request).site_name
+
+    class Link:
+
+        def __init__(self, text, url):
+            self.text = text
+            self.url = url
+
+    def context_404(site_name):
+        if site_name == "Public":
+            return {
+                "extention":
+                "base/public_base.html",
+                "links": [
+                    Link(
+                        "Ask a Librarian",
+                        "https://www.lib.uchicago.edu/research/help/ask-librarian/"
```

Comment:
This should be a site relative link, not absolute. That way it will play nice in our dev environments and on `nest`.